python/pyarrow/tests/test_substrait.py

```diff
@@ -110,12 +111,13 @@ def test_invalid_plan():
     }
     """
     buf = pa._substrait._parse_json_plan(tobytes(query))
-    exec_message = "Empty substrait plan is passed."
+    exec_message = "No RootRel in plan"
```

Comment:
Suggested this for the C++, so it should change here as well. 

```suggestion
    exec_message = "No RelRoot in plan"
```